i18n/de.toml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Why would we need/want to include this file while german is not a supported language?
Same for the other language I don't think we should allow/enable in the conf and ship files for another language but English.
If there is a legit reason for doing this could you please share it?

Reply:
The content file is included as part of the partial update, it was agreed to only keep the english one.